react/src/utils/tableHelpers.ts

```diff
@@ -54,6 +56,7 @@ export const addAdditionalFieldsAndFormatNulls = (
         ...reformatted,
         emptyCaseDetails: '',
         emptyVariationDetails: '',
+        unique_id: currId,
```

Comment:
If you call the parameter you're passing in `unique_id`, you would only need to write `unique_id` in this line versus `unique_id: currId` because JSON understands to take the value of the property with the same name.